src/main/java/de/doubleslash/keeptime/model/Project.java

```diff
@@ -38,6 +39,9 @@
 
    private String name;
 
+   @Lob
+   private String description;

```

Comment:
also add the description field to the toString method (generate it new with eclipse)

Reply:
Done.